backend/vkt/src/main/java/fi/oph/vkt/api/dto/PublicPersonDTO.java

```diff
@@ -1,12 +1,13 @@
 package fi.oph.vkt.api.dto;
 
+import javax.validation.constraints.Size;
 import lombok.Builder;
 import lombok.NonNull;
 
 @Builder
 public record PublicPersonDTO(
   @NonNull Long id,
-  @NonNull String identityNumber,
-  @NonNull String lastName,
-  @NonNull String firstName
+  @NonNull @Size(min = 3, max = 12) String identityNumber,
```

Comment:
Näitä maksimikokoja ei tarvinne kun tä on kai vaan dto joka palautetaan ilmoittautumisen alustamisen yhteydessä backendin päästä